testsuite/MDAnalysisTests/utils/test_streamio.py

```diff
@@ -173,38 +171,44 @@ def create_NamedStream(self, name=None):
         obj = cStringIO()
         return util.NamedStream(obj, name)
 
-    def test_ospath_funcs(self):
-        ns = self.create_NamedStream()
-
+    @pytest.mark.parametrize('funcname', (
+            "abspath",
```

Comment:
Can't we just give the functions as the parameters here?  Ie pass a list of the actual functions rather than the name of the function.  Then for the error message just do `func.__name__`